<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>538</width>
    <height>319</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GoldMiner from Fortune1Coin team (v 0.4)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="labelMineStatus">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>521</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>Miner stoped</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QGroupBox" name="m_groupGPUSelect">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>301</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string>Select your GPU type:</string>
    </property>
    <widget class="QRadioButton" name="radioAMD">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>84</width>
       <height>19</height>
      </rect>
     </property>
     <property name="statusTip">
      <string/>
     </property>
     <property name="text">
      <string>AMD Radeon</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioNVIDIA">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>30</y>
       <width>111</width>
       <height>19</height>
      </rect>
     </property>
     <property name="statusTip">
      <string/>
     </property>
     <property name="text">
      <string>nVIDIA GeForce</string>
     </property>
    </widget>
   </widget>
   <widget class="QLineEdit" name="lineEditEmail">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>8</y>
      <width>261</width>
      <height>22</height>
     </rect>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="statusTip">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="m_startMiningButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>200</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>190</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="text">
     <string>Start mining</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>10</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Enter your e-mail here:</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="m_groupPerfomanceProfile">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>521</width>
      <height>111</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Please pay attention to selecting the performance profile! Your income depends on it! &lt;span style=&quot; font-weight:600;&quot;&gt;The more accurate you specify a performance profile, the better your mining result is.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="title">
     <string>Select your performance profile:</string>
    </property>
    <widget class="QRadioButton" name="m_radioLowProfile">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>501</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Old GPUs and midle notebooks</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="m_radioMidleProfile">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>501</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>GPU  like AMD RX 460, 550, 560, NVIDIA 1050, 1060 series and power laptops</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="m_radioHighProfile">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>501</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>GPU like AMD RX 470, 480, 570, 580, VEGA 56/64. NVIDIA 1070, 1080, 1600 and more powerful</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>538</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuGold_mining_for_games">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="m_actionExit"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="m_actionAbout_this"/>
    <addaction name="m_actionAbout_Fortune1Coin"/>
   </widget>
   <widget class="QMenu" name="menuGo_to_the_site">
    <property name="title">
     <string>Community</string>
    </property>
    <addaction name="m_actionTelegram"/>
    <addaction name="m_actionFacebook"/>
    <addaction name="m_actionVK"/>
    <addaction name="m_actionTwitter"/>
    <addaction name="m_actionReddit"/>
   </widget>
   <addaction name="menuGold_mining_for_games"/>
   <addaction name="menuGo_to_the_site"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="m_actionExit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="m_actionTwitter">
   <property name="text">
    <string>T&amp;witter</string>
   </property>
  </action>
  <action name="m_actionTelegram">
   <property name="text">
    <string>&amp;Telegram</string>
   </property>
  </action>
  <action name="m_actionVK">
   <property name="text">
    <string>&amp;VK</string>
   </property>
  </action>
  <action name="m_actionAbout_this">
   <property name="text">
    <string>About GOLD Mining</string>
   </property>
  </action>
  <action name="m_actionAbout_Fortune1Coin">
   <property name="text">
    <string>About Fortune1Coin</string>
   </property>
  </action>
  <action name="m_actionReddit">
   <property name="text">
    <string>&amp;Reddit</string>
   </property>
  </action>
  <action name="m_actionFacebook">
   <property name="text">
    <string>&amp;Facebook</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
